frontend/packages/console-shared/src/components/editor/ShortcutsLink.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { Popover, Button } from '@patternfly/react-core';
+import { QuestionCircleIcon } from '@patternfly/react-icons';
+import { ShortcutTable, Shortcut } from '../shortcuts';
+
+interface ShortcutsLinkProps {
+  onHideShortcuts?: () => {};
+}
+
+const ShortcutsLink: React.FC<ShortcutsLinkProps> = ({ onHideShortcuts }) => (
+  <div className="ocs-yaml-editor__link">
```

Comment:
This class is controlled by the yaml editor and should not be in the shortcuts link but rather the toolbar renderer.

Reply:
Done